consensus/XDPoS/engines/engine_v2/engine.go

```diff
@@ -376,9 +459,18 @@ func (x *XDPoS_v2) onCountdownTimeout(time time.Time) error {
 }
 
 func (x *XDPoS_v2) broadcastToBftChannel(msg interface{}) {
-	x.BroadcastCh <- msg
+	go func() {
+		x.BroadcastCh <- msg
+	}()
 }
 
 func (x *XDPoS_v2) getCurrentRoundMasterNodes() []common.Address {
 	return []common.Address{}
 }
+
+func (x *XDPoS_v2) getSyncInfo() utils.SyncInfo {
+	return utils.SyncInfo{
```

Comment:
could we change utils.SyncInfo struct from reference type to pointer type, easier for other functions to pass and initiate. 
```
	HighestQuorumCert  QuorumCert
	HighestTimeoutCert TimeoutCert
```
```
	HighestQuorumCert  *QuorumCert
	HighestTimeoutCert *TimeoutCert
```